__tests__/api.test.js

```diff
@@ -45,4 +45,33 @@ describe('api/reviews', ()=>{
             })
```

Comment:
I think I missed this before, but should this have a comment_count property too? 
You can combine the COUNT() aggregate function with AS in your query string to return this in one query